safenode/src/protocol/messages/node_id.rs

```diff
@@ -6,28 +6,33 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
+use crate::network::Error as NetworkError;
```

Comment:
The `protocol` cannot and should not depend on any crate, it has n dependency on any of our crates that implement node/client.